test/cases/compile/error-hide-stack/loader.js

```diff
@@ -3,4 +3,4 @@ module.exports = function() {
 	err.stack = "Stack";
 	err.hideStack = true;
```

Comment:
I'm not sure how these tests work. Can I import `HarmonyLinkingError` here too?

Reply:
You could, but this test is already fine. Loaders won't do use `HarmonyLinkingError` since they probably have own error classes or just add a property.